draft-robert-privacypass-batched-tokens.md

```diff
@@ -53,16 +56,15 @@ scheme defined in {{!AUTHSCHEME=I-D.ietf-privacypass-auth-scheme}}.
 # Motivation
 
 Privately Verifiable Tokens (as defines in
-{{!ISSUANCE=I-D.ietf-privacypass-protocol}}) offer a siple way to unlink the
+{{!ISSUANCE=I-D.ietf-privacypass-protocol}}) offer a simple way to unlink the
 issuance from the redemption. The base protocol however only allows for a single
-token to be issued at a time for every challenge. In some cases, this is not
-sufficient for good unlikability, especially when the redemption happens shortly
-after the issuance. The Batched Token Issuance Protocol allows for multiple aims
-to improve the situation by
-
- - Issuing multiple tokens at a time for a single challenge
- - Making the issuance more efficient by amortizing the cost of the VOPRF proof generation.
+token to be issued at a time for every challenge. In some cases, especially where
+a large number of clients need to fetch a large number of tokens, this may introduce
+performance bottlenecks. The Batched Token Issuance Protocol improves upon the basic
+Privately Verifiable Token issuance protocol in the following key ways:
 
+1. Issuing multiple tokens at once in response to a single TokenChallenge.
+1. Improving server issuance efficiency by amortizing the cost of the VOPRF proof generation.
```

Comment:
Not technically your PR, my wording was not very accurate. The efficiency is also improved on the client side because the proof verification is faster. Also, the size of the proof is not linear in the number of requested tokens anymore.

Reply:
Yep, fixed!